helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -82,11 +82,15 @@ persistence:
   baseSize: 100Gi
 
 postgresql-ams:
-  imageTag: "11.3"
-  postgresUser: alfresco
-  postgresPassword: alfresco
-  postgresConfig:
-    max_connections: 300
+  image:
+    repository: postgres
+    tag: "11.4"
+  postgresqlUsername: alfresco
+  postgresqlPassword: alfresco
+  postgresqlDatabase: postgres
+  postgresqlDataDir: /var/lib/postgresql/data/pgdata
+  postgresqlConfiguration:
+    max_connections: "1500"
```

Comment:
double quotes needed? 
out of curiosity, why did we change it from 300 to 1500?

Reply:
quotes are not needed almost everywhere